tests/test_simulate_from.py

```diff
@@ -882,18 +882,191 @@ def test_population_mismatch_no_population_configs(self):
                 msprime.simulate(from_ts=base_ts, start_time=start_time)
 
 
+class TestSimAncestryInterface:
+    def test_resume_two_population(self):
+        d1 = msprime.Demography()
+        d1.add_population(name="A")
+        d1.add_population(name="B")
+        # All lineages are in B, so we will coalesce
+        ts1 = msprime.sim_ancestry(
+            {"B": 100}, demography=d1, end_time=0.1, random_seed=2
+        )
+        assert ts1.first().num_roots > 1
+
+        ts2 = msprime.sim_ancestry(initial_state=ts1, random_seed=3)
```

Comment:
So, you're proposing this will raise an error, since no population_size is specified, right? That makes sense. But I guess that since there's more than one population (even though only one is used) that the user will need to make a demography object.